acquia/acquia_cms
# ACMS-1069: Update the role permissions for Site Studio access as thes…

…e are not correct.

**Motivation**
Fixes #[ACMS-1069](https://backlog.acquia.com/browse/ACMS-1069)

**Proposed changes**
Add missing permissions(listed below) to Content Administrator, Content Author role:
Site Studio Visual page builder

**Component categories**
- Feature sections YES
- Layout sections YES
- Basic components YES
- Dynamic components YES
- Card components YES
- Slider components YES
- Accordion components YES
- Tab components YES
- Read more components YES
- Map components YES
- Header and footer components NO

**Helper categories**
- Page Layouts YES
- Hero Sections YES
- Text Sections YES
- Card sections YES
- Slider Sections YES
- Accordion sections YES
- Tabbed sections YES
- Dynamic helpers YES
- Layout helpers YES
- Media helpers YES
- Miscellaneous YES

**Alternatives considered**
NA

**Testing steps**
- Login to https://orionacmsdev.prod.acquia-sites.com as administrator
- head towards /admin/people/permissions
- verify listed permissions are enabled for developer role

**Merge requirements**
